api/workAllocation/util.ts

```diff
@@ -809,9 +809,10 @@ export function getStartDate(roleAssignment: RoleAssignment): Date | string {
 }
 
 export function getEndDate(roleAssignment: RoleAssignment): Date | string {
-  if (roleAssignment.roleName === 'specific-access-requested' || roleAssignment.roleName === 'specific-access-denied') {
+  if (roleAssignment.roleName === 'specific-access-requested') {
```

Comment:
Logic is alright but can you add comments / JIRA ref here as it looks a bit unusual. Thanks